.gitignore

```diff
@@ -16,7 +16,7 @@ testbin/*
 cover.html
 
 # folder generated by go mod vendor
-vendor
```

Comment:
Why this change?

Reply:
I removed it when I read source code. I will add it back and rebase on main branch together after https://github.com/redhat-appstudio/integration-service/pull/482 is merged. Thanks!